view/adminhtml/templates/element/exports.phtml

```diff
@@ -0,0 +1,46 @@
+<?php
+/** @var \Magento\Framework\Escaper $escaper */
+/** @var \Hyva\Admin\Block\Grid\Exports $block */
```

Comment:
Please adjust the type hint to be a regular `\Magento\Framework\View\Element\Template`.